product_seeds/data/data.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data noupdate="1">
+    <data noupdate="0">
```

Comment:
if it's 0 then you can remove the attribute as it's the default value

Reply:
I'll put a `fixme` tag and leave it like that while we're developing the module. It's gonna be `1` in the end.